site/en/docs/extensions/mv3/architecture-overview/index.md

```diff
@@ -205,30 +283,28 @@ This method synchronously returns the URL as a `string` and performs no other as
 
 ### More details {: #chrome-more }
 
-For more information, explore the [Chrome API reference docs][31] and watch the following video.
+For more information, explore the [Chrome API reference docs][api-reference].
 
 ## Communication between pages {: #pageComm }
 
-Different components in an extension often need to communicate with each other. Different HTML pages
-can find each other by using the [`chrome.extension`][32] methods, such as `getViews()` and
-`getBackgroundPage()`. Once a page has a reference to other extension pages the first one can invoke
-functions on the other pages and manipulate their DOMs. Additionally, all components of the
-extension can access values stored using the [storage][33] API and communicate through [message
-passing][34].
+Different components in an extension can communicate with each other using [message
+passing][docs-messages]. Either side can listen for messages sent from the other end, and respond on
+the same channel. Additionally, all components of the extension can access values stored using the
+[storage][api-storage] API.
 
 ## Saving data and incognito mode {: #incognito }
```

Comment:
@dotproto My thoughts are that this section touches on how to handle user data in incognito mode, which seems to fit better in [Protect user Privacy](https://developer.chrome.com/docs/extensions/mv3/user_privacy/). What do you think?